.github/workflows/ci.yml

```diff
@@ -345,6 +345,37 @@ jobs:
         run: rush build -o calling
       - name: Tests
         run: rush test -o calling
+      # Not check in the bundles stats if size doesn't change
+      # Stats size in the file is a bit random, so we can't only depend on file itself
+      - name: Check size change
+        id: changescheck
```

Comment:
nit:
changescheck -> compareBundleSize
might be a more indicative name?

Reply:
Remove this step because I refactor the way we do this